src/data/finset/basic.lean

```diff
@@ -2071,6 +2091,9 @@ end finset
 theorem multiset.to_finset_card_le [decidable_eq α] (m : multiset α) : m.to_finset.card ≤ m.card :=
 card_le_of_le (erase_dup_le _)
 
+@[simp] lemma list.card_to_finset [decidable_eq α] (l : list α) :
+  finset.card l.to_finset = l.erase_dup.length := rfl
```

Comment:
I am not immediately convinced that `l.erase_dup.length` is easier to work with than `card l.to_finset`. Could you add an explanation why this is a good `simp` lemma (as a docstring)?